config/routes.rb

```diff
@@ -8,9 +8,6 @@
   root "groups#index", as: "budget"
```

Comment:
- [ ] An **error is thrown** in an attempt to create a category when the _create_ button is pressed. Kindly ensure to debug and improve upon this aspect.

**Error:**
![image](https://user-images.githubusercontent.com/12745474/144693112-88db1f45-b9c2-42e7-975d-768f993acc9c.png)

Reply:
corrected it